ivy/functional/frontends/jax/numpy/statistical.py

```diff
@@ -427,3 +427,17 @@ def ptp(a, axis=None, out=None, keepdims=False):
     x = ivy.max(a, axis=axis, keepdims=keepdims)
     y = ivy.min(a, axis=axis, keepdims=keepdims)
     return ivy.subtract(x, y)
+
+
+@handle_jax_dtype
+@to_ivy_arrays_and_back
+def nanmean(a, axis=None, dtype=None, out=None, keepdims=False, *, where=None):
+    axis = tuple(axis) if isinstance(axis, list) else axis
+    if dtype is None:
+        dtype = "float32" if ivy.is_int_dtype(a) else a.dtype
+    if ivy.is_array(where):
+        where = ivy.array(where, dtype=ivy.bool)
+        a = ivy.where(where, a, ivy.full_like(a, ivy.nan), out=out)
+    ret = ivy.nanmean(a, axis=axis, dtype=dtype, keepdims=keepdims, out=out)
```

Comment:
Hey, I'm pretty sure that nanmean is currently still experimental which means that it isn't going to be available in the newest release. Because of this, I don't think we can call it directly from the frontend. You would need to implement this as a combination of functions in Ivy's functional API instead.

Reply:
Hello James (@jkeane508)
 
I replaced the direct use of nanmean function with the definition of the mean=sum of elements not equal to nan divided by the count of the elements that are not nan